packaging/hydra.nix

```diff
@@ -16,25 +16,16 @@ let
       inherit tarballs;
     };
 
-  testNixVersions = pkgs: client: daemon:
```

Comment:
This was odd. Wrote alternative proposal
- https://github.com/NixOS/nix/issues/11801

Reply:
(to be clear, we agree the way described in the issue is better. This is a step in that direction.)